### PR TITLE
Fixing grid IE compatibility

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?

As title says.

#### What problem is this solving?

Internet Explorer doesn't support grid layout.

#### How should this be manually tested?
[Access this workspace](https://gridfix--storecomponents.myvtex.com/)

#### Screenshots
Before:
captura de tela 20

After:
captura de tela 19
captura de tela 21

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Gallery.js [hunk 1]
```diff
@@ -57,8 +58,9 @@ class Gallery extends Component {
     const from = index * itemsPerRow
     const rowItems = products.slice(from, from + itemsPerRow)
 
-    const containerClasses = classNames(searchResult.galleryRow, {
+    const containerClasses = classNames(searchResult.galleryRow, 'flex', {
       [searchResult.galleryTwoColumns]: layoutMode === 'small' && mobile,
+      'justify-center': mobile,
```
Comment (luciannojunior) [inline react/Gallery.js]: Why only on mobile? The cells are not centered in the desktop version? 
image

Comment (JuanAlmeida12) [reply react/Gallery.js]: Design in web mode aligns left.

**Comment by luciannojunior** [comment]
image

If you visit [adidas.com.br](adidas.com.br) and scroll until the "Destaques" session you can see kind of how our Gallery should behave. There is a center alignment, but the gap between its items are always the same. In larger screens, there will be more free space on both sides (horizontal).

In smaller screens, the item box should decrease its size to **always maintain the same gap** (of course, in the same media query ambient). Flex `justify-center` does just that, excluding the size decrease, what you should accomplish defining the width of the box with relative measures.

File: react/Gallery.js (hunk 1 above)
Comment (theomoura) [reply react/Gallery.js]: We need to involve @oldufre is this discussion to not have conflicting informations about it.

File: react/Gallery.js [hunk 2]
```diff
@@ -58,9 +58,8 @@ class Gallery extends Component {
     const from = index * itemsPerRow
     const rowItems = products.slice(from, from + itemsPerRow)
 
-    const containerClasses = classNames(searchResult.galleryRow, 'flex', {
+    const containerClasses = classNames(searchResult.galleryRow, 'justify-center', {
```
Comment (luciannojunior) [inline react/Gallery.js]: `justify-center` is for flex containers

File: react/Gallery.js [hunk 3]
```diff
@@ -169,7 +168,7 @@ Gallery.propTypes = {
 Gallery.defaultProps = {
   maxItemsPerPage: 10,
   products: [],
-  itemWidth: 300,
+  itemWidth: 285,
```
Comment (luciannojunior) [inline react/Gallery.js]: @oldufre explained that the item width should be dynamic. Has it changed?

**Comment by luciannojunior** [comment]
Include screenshots of the gallery running on a IE browser in desktop, tablet, and mobile mode for at least 4 items

File: react/Gallery.js (hunk 3 above)
Comment (JuanAlmeida12) [reply react/Gallery.js]: Not really, but I will do this in other pr. This is only to fix IE grid compatibility.

**Pull request closed without merging**